kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -367,12 +367,12 @@ def _patch_symbol_table(symbol_table: Dict[str, Any]) -> None:
         KEVM._patch_symbol_table(symbol_table)
 
     @staticmethod
-    def success(s: KInner, dst: KInner, r: KInner, c: KInner) -> KApply:
-        return KApply('foundry_success', [s, dst, r, c])
+    def success(s: KInner, dst: KInner, r: KInner, c: KInner, e1: KInner, e2: KInner) -> KApply:
+        return KApply('foundry_success', [s, dst, r, c, e1, e2])
 
     @staticmethod
-    def fail(s: KInner, dst: KInner, r: KInner, c: KInner) -> KApply:
-        return notBool(Foundry.success(s, dst, r, c))
+    def fail(s: KInner, dst: KInner, r: KInner, c: KInner, e1: KInner, e2: KInner) -> KApply:
+        return notBool(Foundry.success(s, dst, r, c, e1, e2))
```

Comment:
These parameters should have a more descriptive name.  (It's OK if you do not address this in this PR.)